components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/v1/dto/ApplicationTokenGenerateRequestDTO.java

```diff
@@ -26,6 +26,38 @@
     private Long validityPeriod = null;
     private List<String> scopes = new ArrayList<String>();
     private String revokeToken = null;
+
+    @XmlType(name="GrantTypeEnum")
+    @XmlEnum(String.class)
+    public enum GrantTypeEnum {
+        CLIENT_CREDENTIALS("CLIENT_CREDENTIALS"),
+        TOKEN_EXCHANGE("TOKEN_EXCHANGE");
+        private String value;
+
+        GrantTypeEnum (String v) {
+            value = v;
+        }
+
+        public String value() {
+            return value;
+        }
+
+        @Override
+        public String toString() {
+            return String.valueOf(value);
+        }
+
+        @JsonCreator
+        public static GrantTypeEnum fromValue(String v) {
+            for (GrantTypeEnum b : GrantTypeEnum.values()) {
+                if (String.valueOf(b.value).equals(v)) {
+                    return b;
+                }
+            }
+return null;
```

Comment:
is this autogenerated, can't we format this

Reply:
Yes, this is auto generated code